build_run_docker.sh

```diff
@@ -416,10 +387,10 @@ VOLUMES=()
 add_volume_to_template() {
     local VOLUME_NAME="$1"
     local TARGET_DIR_PATH="$2"
-    local VOLUME_TYPE="$3"
-    local IS_NDBD_DATA_DIR="$4"
-    if [ "$VOLUME_TYPE" == "local" ]; then
+    local IS_NDBD_DATA_DIR="$3"
+    if [ "$VOLUME_TYPE" == local ]; then
         local VOLUME_DIR="$LOCAL_VOLUMES_DIR/$VOLUME_NAME"
+        rm -rf "$VOLUME_DIR"
```

Comment:
Might as well just delete all autogenerated files in the beginning?

Reply:
Yup.